packages/web-components/src/horizontalbarchart/horizontalbarchart.styles.ts

```diff
@@ -75,13 +77,54 @@ export const styles: ElementStyles = css`
     opacity: 0;
     z-index: 999;
   }
+  .bar {
+    opacity: 1;
+  }
   .chartTitle {
     display: flex;
     justify-content: space-between;
     text-overflow: ellipsis;
     overflow: hidden;
     white-space: nowrap;
     display: block;
-    color: theme.palette.neutralPrimary;
+    font-size: 14px;
+    font-weight: 400px;
+    line-height: 16px;
+    color: rgb(50, 49, 48);
```

Comment:
instead of hardcoding use the design tokens for here. https://github.com/microsoft/fluentui/blob/master/packages/web-components/src/theme/design-tokens.ts

Reply:
Okay